app/connector/mongodb/pom.xml

```diff
@@ -151,6 +154,11 @@
       <artifactId>hamcrest-library</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-core</artifactId>
```

Comment:
I think we're using AssertJ more than hamcrest in the codebase, I was actually thinking of refactoring hamcrest and replace it with AssertJ

Reply:
Okey, I can give a look and replace with the assertj instead.